loops-and-arrays.js

```diff
@@ -42,3 +52,9 @@ const squares = [30, 2, 8, 24, 11];
 // console.log(numbers) geeft: [30, 2, 8, 24, 11];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [27000, 8, 512, 13824, 1331];
+
+for (let i = 0; i < squares.length; i++) {
+    squares[i] = squares[i] * squares[i] * squares[i];
+}
+
```

Comment:
Het word niet vereist maar je zou in dit geval ook squares[i] = squares[i] ** 3 kunnen gebruiken. 

** betekend tot de macht